### DISCUSSION TITLE
Android 13 working with Termux?

### DISCUSSION THREAD

**Discussion by mg-dev25** [termux/termux-app#3104] [Q&A] [answered]
Hello, I wait for the release of an Android version compatible with Termux, I'm actually on Android 11. They are a warning message for android 12 at the start in the presentation page and noting for Android 13. So I would like to know if Termux work correctly in Android 13? Thanks

**Comment by pressfanksss** [comment, marked as answer]
kind of like yes, but because of the security patch which blocks access to the root folders of Thendoid, naardtli, only if you change the firmware, or wait for patches that may have a hole in the system🤨

**Reply by mg-dev25** [reply]
It is like android 11 and  the need to specify  a working directory or it more restricted?
